### PR TITLE
Fix some log messages that do not match with their function

### PR DESCRIPTION
**Pull request by lzh3636**
## Description

There are some possible copy and paste errors in the log messages (The logging statement was copied from an old place to a new place, but the message wasn't changed to adapt to the function of the new place).  I changed the logging messages to make them more adaptive to record the actual behaviours.

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
Testing
- [ ] I have added tests to cover my changes.
- [ ] All relevant new and existing integration tests have passed.
- [ ] A full integration testsuite with all test that can run on my environment has passed.

### CODE REVIEW

File: engine/orchestration/src/main/java/com/cloud/vm/VirtualMachinePowerStateSyncImpl.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public void resetHostSyncState(long hostId) {
     @Override
     public void processHostVmStateReport(long hostId, Map<String, HostVmStateReportEntry> report) {
         if (s_logger.isDebugEnabled())
```
Comment (rafaelweingartner) [inline engine/orchestration/src/main/java/com/cloud/vm/VirtualMachinePowerStateSyncImpl.java]: Can you remove this IF condition as well?

File: server/src/main/java/com/cloud/resource/ResourceManagerImpl.java [hunk 2]
```diff
@@ -2497,7 +2497,7 @@ public Boolean propagateResourceEvent(final long agentId, final ResourceState.Ev
         }
 
         if (s_logger.isDebugEnabled()) {
```
Comment (rafaelweingartner) [inline server/src/main/java/com/cloud/resource/ResourceManagerImpl.java]: Can you remove this IF check ?
I mean, removing only the IF and maintaining the logging.

Comment (lzh3636) [reply server/src/main/java/com/cloud/resource/ResourceManagerImpl.java]: Yeah, they were done

Comment (rafaelweingartner) [reply server/src/main/java/com/cloud/resource/ResourceManagerImpl.java]: Thanks!

Comment (rafaelweingartner) [reply server/src/main/java/com/cloud/resource/ResourceManagerImpl.java]: Thanks!

**Comment by DaanHoogland** [comment]
travis is ok and i don't see a integration test reason

**Pull request merged**